ghost/admin/app/components/members/modals/bulk-unsubscribe.hbs

```diff
@@ -43,11 +43,28 @@
                     {{#if countFetcher.isLoading}}
                         <GhLoadingSpinner />
                     {{else}}
+                        {{#if this.hasMultipleNewsletters }}
+                         <p>
+                            Which newsletter should these
+                            <strong data-test-text="unsubscribe-count">{{gh-pluralize countFetcher.count "member"}}</strong> be unsubscribed from?
+                        </p>
+                        <OneWaySelect
+                            @value={{this.selectedNewsletterId}}
+                            @options={{this.newsletterList}}
+                            @optionLabelPath="name"
+                            @optionValuePath="value"
+                            @optionTargetPath="value"
+                            @update={{this.setSelectedNewsletter}}
+                            data-test-select="members-single-newsletter-unsubscribe"
+                        />
```

Comment:
Small thing but it's good to always indent html to match the conditionals, helps to keep the code understandable at a glance and avoid problems like we saw in the flag cleanup where the lack of proper indentation made it tricky to see which blocks should be cleaned up.